CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ function(examples list_name)
         get_filename_component(lwe ${l} NAME_WE)
         add_executable(${lwe} ${arpackexample_DIR}/${l} ${examples_EXTRA_SRCS})
         target_link_libraries(${lwe} arpack BLAS::BLAS LAPACK::LAPACK ${EXTRA_LDFLAGS})
-        add_test(NAME "${lwe}_ex" COMMAND ${lwe} WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+        add_test(NAME "${lwe}_ex" COMMAND ${lwe})
```

Comment:
Really?!... That was the only thing to modify to get full build/tests run OK on windows?! 
Can you elaborate/explain a bit why?

Reply:
The search for the DLL is not an absolute path somehow, so it was searching locally for the library. Setting a different working directory meant it was searching for the library there. Since it was absent, it then returned the error code 0xc0000135 indicating DLL not found.